lib/smart_proxy_reports/puppet_processor.rb

```diff
@@ -142,7 +142,7 @@ def spool_report
       payload = measure :format do
         report_hash.to_json
       end
-      SpooledHttpClient.instance.spool(report_id, payload)
+      SpooledHttpClient.instance.spool("report", report_id, payload)
```

Comment:
Please use `:symbols` everytime you want to use a constant in Ruby it's much more efficient. Strings are more expensive.